zerver/forms.py

```diff
@@ -51,7 +51,7 @@
 
 # We don't mark this error for translation, because it's displayed
 # only to MIT users.
-MIT_VALIDATION_ERROR = (
+MIT_VALIDATION_ERROR = Markup(
```

Comment:
Does `DEACTIVATED_ACCOUNT_ERROR` need this, or does `gettext_lazy` handle escaping issues with a username containing HTML?

Reply:
`gettext_lazy` has no variable handling whatsoever; we pass it a literal string and it returns a literal string.

This string is `Markup` because it contains HTML tags. (`Markup` is a subclass of `str` that overrides `+`, `%`, `.format`, `.join` to escape HTML when coercing `str` to `Markup`.) Ordinarily, `Markup` would be interpolated into templates without further escaping. This one, however, is still broken if I remember correctly—it gets double-escaped when wrapped in a `ValidationError`. I’m not fixing that here, only moving the `Markup()` call.

`DEACTIVATED_ACCOUNT_ERROR` is not HTML, so it’s correct for that to be handled as a normal `str` that’s escaped when interpolated into templates in the normal way.